### PR TITLE
Add license GPLv3

### LINKED ISSUE #13: Add license GPLv3

### PR DESCRIPTION
**Pull request by chel-data**
close #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-# ChelFs
+# Chel-FS
 
-A distirbuted filesystem on DAOS object Store.
+## Introduction
+
+Chel-FS is a high performance filesystem built on top of DAOS, a mordern flash medium first storage pool. Chel-FS is aimed for workloads that require high performance filesystem operations. Chel-FS is a layer that is built on top of DAOS. Only filesystem metadata operations will go through Chel-FS, user data will directly flow between DAOS servers and clients. In such a way we can have both high IO bandwidth and IOPS.
```
Comment (josephaug26) [inline README.md:5]: It's a High Performance Distributed Filesystem

Comment (josephaug26) [inline README.md:5]: "Chel-FS is aimed for workloads that require high performance filesystem operations."
>>  High performance and Scale.

Comment (josephaug26) [reply README.md:5]: Scaling is our important feature.

**Review by josephaug26 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chel-data) [reply README.md:5]: What is Scale. Is it scalability? Users may or may not care about scale. I think we don't add scalability. It is inherited from DAOS.

Comment (paul356) [reply README.md:5]: Accepted

Comment (josephaug26) [reply README.md:5]: yes ...If there is not scale then POSIX container is enough. Scalability is important here. Specially Compute Scale.

**Pull request closed without merging**